src/lib/y2storage/planned/partitions_distribution.rb

```diff
@@ -100,6 +100,12 @@ def spaces_total_size
         DiskSize.sum(spaces.map(&:disk_size))
       end
 
+      # Number of assigned spaces in the distribution
+      # @return [Integer]
+      def spaces_count
```

Comment:
is it really worth? I do not see benefit of writting `spaces_count` over `spaces.count`

Reply:
ah, reading it below I see the benefit